packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -441,6 +442,7 @@ export class DataStores implements IDisposable {
 		message: ISequencedDocumentMessage,
 		local: boolean,
 		localMessageMetadata: unknown,
+		addedOutboundReference: (fromNodePath: string, toNodePath: string) => void,
```

Comment:
You can use `this.runtime.addedGCOutboundReference(...)` here. No need to pass this in explicitly.

Reply:
I looked at that too, the main reason I didn't is that this way `detectOutboundReferences` can be a free function which is easier to test.  Also, `this.runtimeaddedGCOutboundReference` takes handles not strings.